### PR TITLE
Application: Replace Stack with Hdy.Deck

### PR DESCRIPTION
**Pull request by danrabbit**
Minimum changes to replace Stack with Deck. This does not include swipe gestures

### CODE REVIEW

File: src/Application.vala [hunk 1]
```diff
@@ -446,15 +445,8 @@ namespace Switchboard {
 
         // Switches to the given plug
         private void switch_to_plug (Switchboard.Plug plug) {
-            if (should_animate_next_transition == false) {
-                stack.set_transition_type (Gtk.StackTransitionType.NONE);
-                should_animate_next_transition = true;
-            } else if (stack.transition_type == Gtk.StackTransitionType.NONE) {
-                stack.set_transition_type (Gtk.StackTransitionType.SLIDE_LEFT_RIGHT);
-            }
-
```
Comment (cassidyjames) [inline src/Application.vala:455]: Removing this means i.e. when you open Network settings from the indicator, that you see the home screen briefly first before it switches to the plug, which feels janky. Is that something you plan to clean up in a follow-up?

Comment (danrabbit) [reply src/Application.vala:455]: Ah hm. Gonna have to dig into this a bit and see if there's a way to temporarily disable animations for Hdy.Deck

**Comment by cassidyjames** [comment]
This seems to do the thing, but when opening to a settings pane directly (i.e. from an indicator), there's a very noticeable "hang" where the window is blank before the settings pane opens. Not sure if that should be considered a blocker, but it's probably something we should still look at.

**Review by cassidyjames — APPROVED** [review]
Actually you know what, that's also in master. This is an improvement. :+1:

**Pull request merged**